.github/workflows/reusable-ubuntu.yml

```diff
@@ -63,13 +63,15 @@ jobs:
         key: ${{ github.job }}-${{ runner.os }}-${{ env.IMAGE_VERSION }}-${{ inputs.config_hash }}
     - name: Configure CPython out-of-tree
       working-directory: ${{ env.CPYTHON_BUILDDIR }}
-      run: ${{ inputs.options }}
+      run: ${{ inputs.options }} CFLAGS="-fdiagnostics-format=json"
     - name: Build CPython out-of-tree
       working-directory: ${{ env.CPYTHON_BUILDDIR }}
-      run: make -j4
+      run: make -j4 &> compiler_output.txt
     - name: Display build info
       working-directory: ${{ env.CPYTHON_BUILDDIR }}
       run: make pythoninfo
+    - name: Check compiler warnings
+      run: python Tools/build/check_warnings.py --compiler-output-file-path=${{ env.CPYTHON_BUILDDIR }}/compiler_output.txt --warning-ignore-file-path ${GITHUB_WORKSPACE}/Tools/build/.warningignore_ubuntu
```

Comment:
No warnings so far, is that correct?

https://github.com/python/cpython/actions/runs/9926876262/job/27421126508?pr=121730#step:18:2

Reply:
Correct. There are no warnings. I tested the script with warnings and in the blog post I linked in the description I show those tests